sdk/loadtestservice/Azure.Developer.LoadTesting/tests/LoadTestAdministrationClientTest.cs

```diff
@@ -0,0 +1,300 @@
+                                "metrics", new Dictionary<string, Dictionary<string, string>>
+                                {
+                                    {
+                                        resourceId, new Dictionary<string, string>
+                                        {
+                                            {"resourceId", resourceId },
+                                            {"metricNamespace", "microsoft.insights/components"},
+                                            {"displayDescription", "sample description"},
+                                            {"name",  "requests/duration"},
+                                            {"aggregation", "Average"},
+                                            {"unit", ""},
+                                            {"resourceType", "microsoft.insights/components"}
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    )
+                );
+        }
+
+        [Test]
+        public async Task GetServerMetricsConfig()
+        {
+            LoadTestAdministrationClient loadTestAdministrationClient = CreateAdministrationClient();
+            await _testHelper.SetupTestingLoadTestResourceAsync(loadTestAdministrationClient, _testId);
+            resourceId = TestEnvironment.ResourceId;
+
+            await loadTestAdministrationClient.CreateOrUpdateServerMetricsConfigAsync(
+                    _testId,
+                    RequestContent.Create(
+                        new Dictionary<string, Dictionary<string, Dictionary<string, string>>>
+                        {
+                            {
+                                "metrics", new Dictionary<string, Dictionary<string, string>>
+                                {
+                                    {
+                                        resourceId, new Dictionary<string, string>
+                                        {
+                                            {"resourceId", resourceId },
+                                            {"metricNamespace", "microsoft.insights/components"},
+                                            {"displayDescription", "sample description"},
+                                            {"name",  "requests/duration"},
+                                            {"aggregation", "Average"},
+                                            {"unit", ""},
+                                            {"resourceType", "microsoft.insights/components"}
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    )
+                );
+
+            Response response = await loadTestAdministrationClient.GetServerMetricsConfigAsync(_testId);
+            Assert.NotNull(response);
```

Comment:
check for correct metrics?

Reply:
will add deeper asserts with the latest commit